ledger-core/insolar/payload/proto_vnvn.r0.proto

```diff
@@ -108,11 +108,11 @@ message VStateReport {
     option (insproto.id) = 1004;
 
     enum StateStatus {
-        Unknown = 0; // zero value in Go
-        Ready = 1;
-        Empty = 2;
-        Inactive = 3;
-        Missing = 4;
+        StateStatusInvalid = 0; // zero value in Go
```

Comment:
maybe `StateReportStatusXxx` ?

Reply:
That PR is not about renaming types, but renaming constants. We can rename that type in separate PR

Reply:
ok